docs/development-workflow.md

```diff
@@ -51,6 +51,18 @@ or limit it as before with:
 DEBUG=calypso:* make run
 ```
 
+## Clearing browser data
+
+Seeing some strange errors that only happen on _your_ browser? You may need to clear your browser's local storage and database. Calypso saves a lot here and if you've changed the way your data is kept (especially in redux state), it can break things in unepxected ways during the transition.
```

Comment:
> fixing strange issues that arise when the code and stored data are no longer compatible.

Our schemas should be protecting us from this case, even when switching branches. Anything in particular that you ran into? That's likely a smell that something is being persisted that is missing a schema, or shouldn't be. See also: https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#problem-data-shapes-change-over-time--3101-